src/Uno.UI/DataBinding/BindingPropertyHelper.cs

```diff
@@ -328,7 +331,7 @@ private static PropertyInfo GetPropertyInfo(Type type, string name)
 					BindingFlags.Instance |
 					BindingFlags.Static |
 					BindingFlags.Public |
-					BindingFlags.DeclaredOnly
+					BindingFlags.NonPublic
```

Comment:
Does this mean that bindings in general are now going to (incorrectly) bind to non-public properties?

Reply:
Indeed, and DeclaredOnly should still be here. I'll add some new tests,